docs/index.rst

```diff
@@ -3,32 +3,65 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to Viper's documentation!
-=================================
-Viper is an experimental programming language that aims to provide the following features:
+#####
+Viper
+#####
 
-* Bounds and overflow checking, both on array accesses and on arithmetic
-* Support for signed integers and decimal fixed point numbers
-* Decidability - it's possible to compute a precise upper bound on the gas consumption of any function call
-* Strong typing, including support for units (eg. timestamp, timedelta, seconds, wei, wei per second, meters per second squared)
-* Maximally small and understandable compiler code size
-* Limited support for pure functions - anything marked constant is NOT allowed to change the state
+Viper is an **experimental**, contract-oriented, and high-level programming language whose syntax is similar to Python and it is
+designed to target the Etherum Virtual Machine (EVM).
 
```

Comment:
How do you think this sounds?
Viper is an **experimental**, contract-oriented, pythonic programming language that is designed to run on the Etherum Virtual Machine (EVM).

Reply:
@DavidKnott Yes definitely better. It's just adapted from the Solidity docs which isn't ideal in this case. I think I'll add a link to ethdocs.org as well.

Reply:
:+1: